features/donations/quick_donation.feature

```diff
@@ -5,6 +5,11 @@ Feature: quick donation without logging in
   To make it easier for people to donate
   I want to provide a way to donate with a credit card without logging in
 
+Background:
+  Given the following account codes exist:
+  | name             | code | description                                                     | donation_prompt          |
+  | Soda Fund        | 0504 | The Soda Funds aims to put a Soda Fountain in Soda Hall         |                          |
```

Comment:
not a big deal, but in general it is better to stick to relevant and/or existing fake data for test cases, since the tests are supposed to serve as documentation.